resolwe/flow/executors/__init__.py

```diff
@@ -140,6 +147,16 @@ def run(self, data_id, script, verbosity=1):
 
                         for obj in iterjson(line[3:].strip()):
                             spawn_processors.append(obj)
+                    elif line.strip().startswith('export'):
+                        file_name = line[6:].strip()
+
+                        export_folder = settings.FLOW_EXECUTOR['UPLOAD_DIR']
+                        export_path = os.path.join(export_folder, str(data_id))
```

Comment:
I do not like using data IDs. I would rather rename the files to some random UUID name when we move them to the upload dir.